RocketChat/Rocket.Chat.ReactNative
# Regression: right icons overlapping when users name is long

## Proposed changes
In this PR #4048  we created an edited icon and aligned all messages icons on the right. 
But when the user name is long, the icons was overlapping each others. So the proposal of this PR is fixing this wrong behavior.

## Issue(s)	


## How to test or reproduce
We need to test all messages icons, for do that we need:
* test edited messages;
* test messages with errors;
* test encrypted messages;
* test read receipt messages;

Keep in mint to test with users with long and short names. And remember to test all things listed above with messages header and without header.
Also test the main features from messages, just to make sure everything stills work fine. 
(E.G: send message, send audio, share images...)

## Screenshots
| BEFORE | AFTER |
|---------|--------|
|![Screen Shot 2022-05-26 at 09 56 55](https://user-images.githubusercontent.com/14894380/170548323-c82c209b-f2b0-437b-87e0-00b7f72ab813.png)|![Screen Shot 2022-05-26 at 15 04 33](https://user-images.githubusercontent.com/14894380/170548807-8518632d-96e3-4ec2-8401-4d18df8f397f.png)|


## Types of changes




## Checklist



## Further comments
